src/devices/Bh1745/samples/Bh1745.Sample.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Bh1745.Samples
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // bus id on the raspberry pi 3
+            const int busId = 1;
+
+			// create device
```

Comment:
looks like you have some tabs in your .cs and .csproj files, do you mind changing all of them to spaces instead?

Reply:
yeah seems like I didn't check the files before comitting, shame on me 😄 Should be fixed